src/Symfony/Component/Validator/Constraints/IsJsonValidator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * @author Imad ZAIRIG <imadzairig@gmail.com>
+ */
+class IsJsonValidator extends ConstraintValidator
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!$constraint instanceof IsJson) {
+            throw new UnexpectedTypeException($constraint, __NAMESPACE__.'\IsJson');
+        }
+
+        if (!$this->is_json($value)) {
```

Comment:
Normally, these validations do an early exit if the value is `null` to allow for optional fields.

Reply:
Sorry can you give me an example ? Do you mean that if the value is null , we should skip this test ? 

Thanks